app/views/adoption_mailer/interest_email.html.erb

```diff
@@ -1 +1,9 @@
-Your <%= @thing %>, <%= @name %>, which is currently up for adoption, got some interest today.  <%= mail_to @email %> is interested in adopting it. Please get in touch with them.
+Your <%= @thing %>, <%= @name %>, which is currently up for adoption, got some interest today.<br />
+<%= link_to(@name, "#{Rails.root}#{cookbook_path(@name)}") %>
+<br />
+<br />
+Here is the interested user's information:<br />
```

Comment:
Here is the information you need to know about the awesome Chef who is interested in adopting your <%= @thing %>.